index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1>"Hello World, we are software developers!"</h1>
```

Comment:
I think you should use an HTML tag such as <header></header> to give the website a position of logos, navigation bar, etc

Reply:
- [ ] Hello @migo1 Good job for suggest the changes by mentioning the how to implemented the suggested change but here your suggestion is not fully mentioned to to student ,  "I think you should use an HTML tag such as ................(????)
to give the website a position of logos, navigation bar, etc", So kindly Fix it.

Reply:
I think you should use an HTML tag such as <header></header> tag it contains introductory content, such as a logo, website title, navigation menu, and other relevant information about the website.